models/hashflow/avalanche_c/hashflow_avalanche_c_crosschain_trades.sql

```diff
@@ -0,0 +1,103 @@
+{{
+    config(
+        alias="avalanche_c_cross_chain_trades"
```

Comment:
Since the schema is `hashflow_avalanche_c`,
we can just have alias as `cross_chain_trades` here.

The same goes to other base models as well.